test/attr/attr_expose.swift

```diff
@@ -33,3 +33,17 @@ struct ExposedStruct {
     @_expose(Cxx, "initWith")
     init(with y: Int) {}
 }
+
+@_expose(wasm) func exposeToWasm() {}
+@_expose(wasm, "with_name") func wasmName() {}
+@_expose(wasm, "") func wasmEmptyNameOk() {}
+
+@_expose(wasm) struct WasmErrorOnStruct { // expected-error {{@_expose attribute with 'wasm' can only be applied to global functions}}
+  @_expose(wasm) func errorOnMethod() {} // expected-error {{@_expose attribute with 'wasm' can only be applied to global functions}}
+}
+
+func wasmNested() {
+  @_expose(wasm) // expected-error {{attribute '_expose' can only be used in a non-local scope}}
+  func errorOnInnerExpose() {}
+}
+
```

Comment:
Could you please add a test with multiple `@_expose` attributes on a single decl?